data/json/furniture_and_terrain/furniture-tools.json

```diff
@@ -292,6 +292,27 @@
     "flags": [ "TRANSPARENT", "NOITEM", "EASY_DECONSTRUCT" ],
     "bash": { "str_min": 4, "str_max": 8, "sound": "crunch!", "sound_fail": "whump.", "items": [ { "item": "still", "count": 1 } ] }
   },
+  {
+    "type": "furniture",
+    "id": "f_screw_press",
+    "name": "screw press",
+    "description": "A large screw press for the purpose of manually pressing oil out of fruit or seed paste.  The paste gets placed in wetted bags and gets pressed down by the force of the screw mechanism.",
+    "symbol": "T",
+    "color": "light_red",
+    "move_cost_mod": -1,
+    "coverage": 40,
+    "required_str": -1,
+    "crafting_pseudo_item": "screw_press",
+    "deconstruct": { "items": [ { "item": "screw_press", "count": 1 } ] },
```

Comment:
this hsould give back the item from the construction recipe, also `screw_press` is defined as a pseudo item, I guess what you want here is `screw_press_tool`

Reply:
Changed it to give back screw_pres_tool now.